### ISSUE TITLE
TS2349 [ERROR]: This expression is not callable.

### ISSUE THREAD

**Issue by j-o-o** [RubenArtmann/bandcamp-dl#4]
During the installation I stumble across this error:
```
Check https://raw.githubusercontent.com/RubenArtmann/bandcamp-dl/main/mod.ts
error: TS2349 [ERROR]: This expression is not callable.
  Type '{}' has no call signatures.
		string = slug(string,{
		         ~~~~
    at https://raw.githubusercontent.com/RubenArtmann/bandcamp-dl/main/mod.ts:36:12

TS2339 [ERROR]: Property 'decode' does not exist on type '{}'.
				track = sanitizeFileName(he.decode(optionalOverwrites[1]));
				                            ~~~~~~
    at https://raw.githubusercontent.com/RubenArtmann/bandcamp-dl/main/mod.ts:136:33

TS2339 [ERROR]: Property 'decode' does not exist on type '{}'.
				if(optionalOverwrites[2]) album = sanitizeFileName(he.decode(optionalOverwrites[2]));
				                                                      ~~~~~~
    at https://raw.githubusercontent.com/RubenArtmann/bandcamp-dl/main/mod.ts:137:59

TS2339 [ERROR]: Property 'decode' does not exist on type '{}'.
				artist = sanitizeFileName(he.decode(optionalOverwrites[3]));
				                             ~~~~~~
    at https://raw.githubusercontent.com/RubenArtmann/bandcamp-dl/main/mod.ts:138:34

Found 4 errors.
```
I'm on Mac OS Catalina

**Comment by RubenArtmann** [comment]
all the imports seem to get silently ignored on the newest version of deno:
```
import he from "https://jspm.dev/npm:he@1.2.0!cjs";
import slug from "https://jspm.dev/limax";
```

for now you can just downgrade: 
```
$ deno upgrade --version="1.4.6"
```

**Comment by RubenArtmann** [comment]
it seems like them defaulting to isolatedModules caused this.
for now add ```--no-check``` (i updated the readme)

**Comment by RubenArtmann** [comment]
fixed completely in [4ae9ef0e](https://github.com/RubenArtmann/bandcamp-dl/commit/4ae9ef0ec7bdf7ae94f987c69b7ce100044dfefa)
